pmd-lua/src/main/antlr4/net/sourceforge/pmd/lang/lua/antlr4/Lua.g4

```diff
@@ -234,6 +245,58 @@ string
     : NORMALSTRING | CHARSTRING | LONGSTRING
     ;
 
+SimpleType
```

Comment:
These are lexer rules... is that on purpose? The issues I pointed out would not show up were this a parser rule I think

Reply:
derp. fixed.